From 79c3c1ab7faee8247b740b4ec660a998f2881633 Mon Sep 17 00:00:00 2001
From: V8 Autoroll <v8-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Date: Fri, 07 Feb 2025 12:26:19 -0800
Subject: [PATCH] Version 13.3.415.21

Version incremented at https://cr-buildbucket.appspot.com/build/8723562094979003089

Change-Id: Iceb76efcc2e405a476b4e48c103543dbacb7a5f0
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/6244840
Bot-Commit: v8-ci-autoroll-builder <v8-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/13.3@{#41}
Cr-Branched-From: 41dacffe436aeb9311879cb07648f1e36609a804-refs/heads/13.3.415@{#1}
Cr-Branched-From: 3348638c0af67c885b30891a358c89a917ac9759-refs/heads/main@{#97937}
---

--- a/v8/include/v8-version.h
+++ b/v8/include/v8-version.h
@@ -11,7 +11,7 @@
 #define V8_MAJOR_VERSION 13
 #define V8_MINOR_VERSION 3
 #define V8_BUILD_NUMBER 415
-#define V8_PATCH_LEVEL 19
+#define V8_PATCH_LEVEL 21
 
 // Use 1 for candidates and 0 otherwise.
 // (Boolean macro values are not supported by all preprocessors.)
